package.json

```
@@ -14,15 +14,12 @@
     "charts component"
   ],
   "license": "MIT",
-  "main": "dist/vue-google-charts.common.js",
+  "main": "dist/vue-google-charts.cjs.js",
```

```suggestion
  "main": "dist/vue-google-charts.cjs",
```

Need to use `.cjs` extension. It's important for [conditional exports](https://nodejs.org/api/packages.html#conditional-exports)